examples/Dockerfile.exhaustive

```diff
@@ -21,7 +21,7 @@ ENV chse_4=value4 chse_5="value5 foo" chse_7=\"value7\"
 #ENV chse_4=value4 chse_5="value5 foo" chse_6=value6\ foo chse_7=\"value7\"
 
 RUN pwd
-WORKDIR /usr/local/src
+WORKDIR /usr/src
```

Comment:
A lot of the examples are changed to use prefix `/usr` instead of `/usr/local`. Why is that?

Reply:
In CentOS the linker doesn't search `/usr/local/lib` by default like Debian does so there were two options that I saw:
1. Add `/usr/local` to search path by adding something like `/etc/ld.so.conf.d/usrlocal.conf`
2. Move all of the examples to `/usr` instead

When we previously discussed this (a long time ago) we agreed that it was probably best to just move the examples.

Reply:
Aha. I wonder now if that was the right decision. For example, on another image I'm installing GCC from source into `/usr/local`. It seems to cause a bit of a clash with the system GCC, but at least it's in a separate place and doesn't overwrite a random subset of the system GCC. With the new GCC in `/usr/local`, I could in principle delete the system GCC, and it wouldn't interfere with the new one.

Reply:
I'll change back to `/usr/local/` and add that location to the search path.